Gemfile

```diff
@@ -23,7 +23,7 @@ gem "sprockets-rails"
 gem 'sprockets_uglifier_with_source_maps'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer',  platforms: :ruby
-gem 'puma', '~> 5.6.8'
+gem 'puma', '~> 6.4.2'
```

Comment:
question: Is this intentional? Seems unusual to fold a major version ruby gem change into a Node package update.

Reply:
Thank you for calling that out! This snuck in because I was getting a bunch of bundler issues with 5.6.8, upgrading to 6.4.2 solved those (and brought us in line with Partners). I extracted this into its own PR (#2073) so the separation is clearer